fastlane_bot/helpers/txhelpers.py

```diff
@@ -798,9 +803,9 @@ def approve_token_for_arb_contract(self, token_address: str, approval_amount: in
             else:
                 return None
 
-    def _get_layer_one_gas_fee_loop(self, rawTransaction) -> Decimal:
+    def _get_layer_one_gas_fee_1(self, rawTransaction) -> Decimal:
```

Comment:
This change can be reverted, as it will be obtained directly from a similar task already completed for Base and Optimism (which would otherwise lead to manual conflict when this change is merged).